### PR TITLE
ST-432-config-for-sptribs-case-api

### PR DESCRIPTION
**Pull request by geciciserkan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ST-432
https://tools.hmcts.net/jira/browse/ST-378

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: apps/sptribs/aat/01/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: nfdiv
+resources:
+  - ../base
+patchesStrategicMerge:
+  - ../../nfdiv-cron-eligible-for-switch-to-sole/aat/01.yaml
```
Comment (adusumillipraveen) [inline apps/sptribs/aat/01/kustomization.yaml:7]: nfdiv? can u pls get this reviewed internally first.

File: k8s/namespaces/rd/rd-professional-api/aat.yaml [hunk 2]
```diff
@@ -8,5 +8,5 @@ spec:
       environment:
         DELETE_ORG: false
         ACTIVE_ORG_EXT: true
-        PRD_S2S_AUTHORISED_SERVICES: rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,finrem_case_orchestration,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend,nfdiv_case_api,am_org_role_mapping_service,probate_backend,civil_service,payment_app,prl_cos_api,civil_general_applications
```
Comment (kavithagade12) [inline k8s/namespaces/rd/rd-professional-api/aat.yaml:11]: @geciciserkan : Is there any requirement to whitelist special tribunals from professional reference data. If so, please let us know. For production whitelisting you have to go through the release process.

**Review by spotstory — CHANGES_REQUESTED** [review]
files k8s/namespaces/rd/rd-professional-api/demo.yaml , ithc.yaml, perftest.yaml appear to have the sptribs_casi_api mentioned twice on the one row

File: k8s/namespaces/rd/rd-professional-api/aat.yaml (hunk 2 above)
Comment (geciciserkan) [reply k8s/namespaces/rd/rd-professional-api/aat.yaml]: I am not sure if we need it. I removed for now. I will add if we need it in the future

**Review by spotstory — DISMISSED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM, don't see any issues

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]
Hi, this has too many irrelevant changes. all new apps should be configured using  https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md

**Comment by geciciserkan** [comment]
we created following PR instead

https://github.com/hmcts/cnp-flux-config/pull/18037

**Pull request closed without merging**